### ISSUE TITLE
CLI doesnt work

### ISSUE THREAD

**Issue by pmelisko** [andyhu/transliteration#25]
Hi, I have problem with CLI version

```
_npm install --save transliteration_

slugify "Štandardy pre informačné systémy verejnej správy"

module.js:457
    throw err;
    ^

Error: Cannot find module '../../build/node/utils'
    at Function.Module._resolveFilename (module.js:455:15)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/pmelisko/Documents/workspace/init/node_modules/transliteration/lib/bin/slugify:8:14)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
```

build folder doesn't exist
after fixing of requires to proper paths in lib/bin/slugify (similarly in lib/bin/transliterate) it will work

```
var _utils = require('../../build/node/utils'); -> var _utils = require('../node/utils');
var _slugify = require('../../build/node/slugify'); -> var _slugify = require('../node/slugify');
```

```
_slugify "Štandardy pre informačné systémy verejnej správy"_
standardy-pre-informacne-systemy-verejnej-spravy
```

PS: is there any possibility to read input from stdin? something like:
$ echo "Štandardy pre informačné systémy verejnej správy" | slugify
Invalid argument. Please type 'node_modules/.bin/slugify --help' for help.
undefined

**Comment by andyhu** [comment]
Hi @pmelisko,
Thanks for the spot. The incorrect include path was fixed in version 1.2.4.
Regarding to your question about using stdin as input, I'll consider it a bit and probably get it work in next version.
Cheers!

**Comment by andyhu** [comment]
@pmelisko Stream input is now supported in version 1.3.0. You can use it as follow:
`$ echo "Štandardy pre informačné systémy verejnej správy" | slugify -S`
